old/main.js

```diff
@@ -384,12 +384,12 @@ function initWindow(window) {
           equipments
             .map(
               ([name, icon]) =>
-                `<img src='${icon}' class='currencyText'/><span class='eventText'>${name}</span>`
+                `<span><img src='${icon}' class='currencyText'><span class='eventText'>${name}</span></span>`
             )
             .join('; '),
         true
       );
-    }
+    }``
```

Comment:
Should this be here?

Reply:
hmm didn't notice that, but as this file was obsoleted I guess it was fine. Will just revert this file